### PR TITLE
New package: acme.sh-3.0.2

### PR DESCRIPTION
**Pull request by Wesley-Chan**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by Wesley-Chan** [comment]
Hello, @ericonr. Would you please help review this PR or assign it to one of the maintainers? Thank you.

**Comment by Chocimier** [comment]
Should be named with a dot: `acme.sh`

**Comment by Wesley-Chan** [comment]
Thanks for your reply, @Chocimier. I've changed the name to `acme.sh`.

File: srcpkgs/acme.sh/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+license="GPL-3.0-or-later"
+homepage="https://github.com/acmesh-official/acme.sh"
+distfiles="https://github.com/acmesh-official/acme.sh/archive/refs/tags/${version}.tar.gz"
+checksum=6212cc0c2bca99a7dd6cbb4236b4c7dd5d1113dab0841e66dae4d307d902a8e6
+
+do_install() {
+	vmkdir usr/share/acme.sh/deploy 755
+	vmkdir usr/share/acme.sh/dnsapi 755
+	vmkdir usr/share/acme.sh/notify 755
+	vmkdir usr/bin 755
```
Comment (paper42) [inline srcpkgs/acme.sh/template:17]: 755 should be the default

File: srcpkgs/acme.sh/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'acme.sh'
+pkgname=acme.sh
+version=3.0.1
+revision=1
+wrksrc="acme.sh-${version}"
```
Comment (paper42) [inline srcpkgs/acme.sh/template:5]: this line is not required now

Comment (Wesley-Chan) [reply srcpkgs/acme.sh/template]: Thanks, @paper42.

**Comment by Chocimier** [comment]
please squash into `New package: acme.sh-3.0.2`

**Comment by Wesley-Chan** [comment]
Hello, @Chocimier and @paper42. Could you please take some time to review this? Thank you very much.

**Pull request merged**